windows/benji.py

```diff
@@ -72,7 +73,19 @@ def events(frame,put):
 	pc_locations = ("desktop", "documents", "downloads")
 	
 	put = put.lower()
-	link = put.split()
+	translator = Translator()
+	if put.startswith("spanish"):
+		word, space, rest = put.partition(' ')
+		translated = translator.translate(rest, src='es', dest='en')
+		put = translated.text
+		link = put.split()
+	elif put.startswith("french"):
+		word, space, rest = put.partition(' ')
+		translated = translator.translate(rest, src='fr', dest='en')
+		put = translated.text
+		link = put.split()
+	else:

```

Comment:
since `link=put.split()` is happening in all 3 cases, you put that statement outside the if-else condition, and get rid of the extra else condition.